extension/src/test/suite/completionItemDcl.test.ts

```diff
@@ -0,0 +1,58 @@
+import * as path from 'path';
+import { assert, expect } from 'chai';
+import { Position, CompletionContext } from "vscode";
+import { ReadonlyDocument } from '../../project/readOnlyDocument';
+import { invokeCompletionProviderDcl } from "../../completion/completionProviderDcl";
+import { CompletionItemDcl, Kinds } from "../../completion/completionItemDcl";
+import { AutoLispExt } from '../../context';
+
+
+const extRootPath = path.resolve(__dirname, '../../../');
+const symbolsFileTest = path.resolve(extRootPath, "./extension/src/test/SourceFile/test_case/symbols.lsp");
+const commentsFileTest = path.resolve(extRootPath, "./extension/src/test/SourceFile/test_case/comments.lsp");
+//const markupsFileTest = path.resolve(extRootPath, "./extension/src/test/SourceFile/test_case/pdfMarkups.lsp");
+const largeFileTest = path.resolve(extRootPath, "./extension/src/test/SourceFile/unFormatted10.lsp");
+
+
+// NOTE: Just because these tests return results and pass, manual testing is still required. VSCode can be given
+//		 whatever we want and still choose to kick them out from the users perspective under various conditions.
+
+suite("Parsing: DocumentContainer Tests", function () {	
```

Comment:
This was an early test fixture. The entire file needs to be deleted. The tests are all inside `provider.completionDcl.test.ts` now.